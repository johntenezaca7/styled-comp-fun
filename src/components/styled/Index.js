export { Button } from "./Elements/Button";
export { Title } from "./Elements/Title";
export { Section } from "./Elements/Section";
export { FlexContainer } from "./Elements/FlexContainer"
export { Input } from "./Elements/Input";
export { PrimaryTheme } from "./Elements/PrimaryTheme";
export { GlobalStyles } from "./Elements/GlobalStyles";
