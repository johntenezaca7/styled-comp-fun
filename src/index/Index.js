import React from "react";
import { render } from "react-dom";
import Init from "./Init";

render( <Init /> , document.getElementById("root"));