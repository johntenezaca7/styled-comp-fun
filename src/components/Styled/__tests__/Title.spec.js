import React from "react";
import renderer from "react-test-renderer";
import 'jest-styled-components';

import { Title } from "../Index";

test("Button renders correctly.", () => {
  const element = renderer.create(<Title />).toJSON();
  expect(element).toMatchSnapshot();
});