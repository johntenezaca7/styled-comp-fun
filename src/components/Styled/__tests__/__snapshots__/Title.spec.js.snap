// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly. 1`] = `
.c0 {
  color: palevioletred;
  font-size: 2.5rem;
  margin-top: 0;
  margin-bottom: 12px;
}

<h1
  className="c0"
/>
`;
